x-pack/plugin/sql/qa/server/src/main/resources/datetime.csv-spec

```diff
@@ -651,6 +651,108 @@ HAVING DATETIME_FORMAT(MAX(birth_date), 'dd')::integer > 20  ORDER BY 1 DESC;
 1961-02-26 00:00:00.000Z | 02
 ;
 
+selectDateFormat
+schema::df_date:s|df_datetime:s|df_time:s
```

Comment:
It would be nice to have some tests that cover patterns that contain random strings, eg. 

```
DATE_FORMAT(some_date, 'he was born on the %dth of %M')
```